sopel/modules/url.py

```diff
@@ -353,16 +357,30 @@ def title_auto(bot: SopelWrapper, trigger: Trigger):
         bot.memory["last_seen_url"][trigger.sender] = url
 
 
+class UrlInfo(NamedTuple):
```

Comment:
Do we have a compelling reason not to call this `URLInfo`? We already have e.g. `sopel.plugins.rules.URLCallback` as precedent for not forcing strict CamelCase on acronyms/initialisms.

Reply:
Oh yeah, `URLInfo` would be better.